src/test/java/com/epam/trn/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,147 @@
+package com.epam.trn.hw2.ex1;
+
+import com.epam.trn.hw2.AbstractTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Exercise1 extends AbstractTest {
+
+    @Test
+    public void exercise1() {
+        // Step 1: Open test site by URL
+        // Data: "https://jdi-testing.github.io/jdi-light/index.html"
+        // Exp. result: Test site is opened
+
+        String URL = "https://jdi-testing.github.io/jdi-light/index.html";
+
+        webDriver.get(URL);
+        softAssert.assertEquals(webDriver.getCurrentUrl(), URL);
+
+        // Step 2: Assert browser title
+        // Data: "Home page"
+        // Exp. result: Browser title equals "Home Page"
+
+        softAssert.assertEquals(webDriver.getTitle(), "Home Page");
+
+        // Step 3: Perform login
+        // Data: username: Roman, pass: Jdi1234
+        // Exp. result: User is logged
+
+        String login = "Roman";
+        String pass = "Jdi1234";
+
+        webDriver.findElement(By.id("user-icon")).click();
+        webDriver.findElement(By.xpath("//input[@id='name']")).sendKeys(login);
+        webDriver.findElement(By.id("password")).sendKeys(pass);
+        webDriver.findElement(By.id("login-button")).click();
+        softAssert.assertTrue(webDriver.findElement(By.className("logout")).isDisplayed());
+
+        // Step 4: Assert username is logged
+        // Data: ROMAN IOVLEV
+        // Exp. result: Name is displayed and equals to expected result
+
+        String name = "ROMAN IOVLEV";
+
+        WebElement loggedUser = webDriver.findElement(By.id("user-name"));
+        softAssert.assertTrue(loggedUser.isDisplayed());
+        softAssert.assertEquals(loggedUser, name);
+
+        // Step 5: Assert that there are 4 items on the header section are displayed and they have proper texts
+        // Data: "HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"
+        // Exp. results: Menu buttons are displayed and have proper texts
+
+        List<WebElement> buttons = webDriver.findElements(By.xpath("//ul[contains(@class, 'm-l8')]/li"));
```

Comment:
```suggestion
        List<WebElement> buttons = webDriver.findElements(By.css("ul.m-l8 > li"));
```
Same for other locators